.travis.yml

```diff
@@ -52,3 +52,9 @@ deploy:
   on:
     tags: true
     condition: $DEPLOY_ARTIFACTS = 1
+
+notifications:
+  slack:
+    rooms:
+      - secure: "4NsPGDKKtiUSdqjl3bxUMlbw"
```

Comment:
Really minor nit: since the hyphens aren't indented in the rest of this file, you probably want to unindent this one level.